flipperdevices/Flipper-Android-App
# Move more filemngr modules to mpp

**Background**

Some modules of new file manager can be moved to multiplatform, so here's this transition

**Changes**

- Move all filemngr modules to MPP (except upload - it depends on deeplinks)
- Move coil to MPP with coil3 & ktor3 for server-image loading

**Test plan**

- See CI is ok
- Open flipper app and see images loading
- Open new file manager see it's working
